### DISCUSSION TITLE
目录公开访问权限

### DISCUSSION THREAD

**Discussion by lori255** [messense/aliyundrive-webdav#542] [Ideas]
如果配置某个或多个目录为公开目录，即配置的目录访问时不需要账户验证，其它目录还是需要验证。
例如：
配置 /a/** 目录为公开

访问 /a/aa/aaa.txt 和 /a/bb/bbb.txt 都不需要权限
访问 /c/** 则需要权限

**Comment by messense** [comment]
没兴趣。
